packages/remix-validated-form/src/ValidatedForm.tsx

```diff
@@ -329,7 +329,7 @@ export function ValidatedForm<DataType>({
       // If we use `event.currentTarget` here, it will break because `currentTarget`
       // will have changed since the start of the submission.
       if (fetcher) fetcher.submit(submitter || e.currentTarget);
-      else submit(submitter || target, { method, replace });
+      else submit(submitter || target, { replace });
```

Comment:
It looks like the corresponding code in the remix repo still passes `method`. Does the expected behavior reproduce when using `Form` from `remix`?

https://github.com/remix-run/remix/blob/dev/packages/remix-react/components.tsx#L966

Reply:
Good catch @airjp73, you're right Remix's still passes the `method` and no; the expected behavior currently doesn't work on Remix's side. There's a pending PR about this here which apply a similar patch https://github.com/remix-run/remix/pull/2162

Reply:
Thanks for the extra context! The explanation in the commit message is nice, too. :)